prisma/schema.prisma

```diff
@@ -308,6 +308,8 @@ model InventoryCountTemplate {
 
   branch                         RestaurantBranch                @relation(fields: [branchId], references: [id])
   inventoryCountTemplateStorages InventoryCountTemplateStorage[]
+
+  @@unique([type, branchId])
```

Comment:
should not be unique, because we can have 
2 daily templates, but different storages like fridge and chemicals. first one is checked by cooks, second one is checked by cleaners